src/amuse/support/testing/nbody/nbody_tests.py

```diff
@@ -0,0 +1,771 @@
+from dataclasses import make_dataclass
```

Comment:
This is unused, and so are `exception`, `time`, 'OrderedDictionary' and 'fixture', and `pytest` is imported twice.